Makefile

```diff
@@ -113,12 +118,26 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	make -C conmon clean
 	make -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+local-cross:
+	@$(MAKE) $(CROSS_BUILD_TARGETS); \
+	for file in $(CROSS_BUILD_TARGETS); do test -s $$file && exit 1 ; done
+
+.PHONY: bin/crio.cross.%
+bin/crio.cross.%: .gopathok $(shell hack/find-godeps.sh $(GOPKGDIR) cmd/crio $(PROJECT))
+	@rm -f $@.failed
+	@mkdir -p bin
```

Comment:
Do you need this?  I think `$(GO) build ...` will do this for you (at least, `rm -rf bin && make bin/crio` works for me in master, and it is not making `bin` explicitly).

Reply:
ah, i guess cruft. I had had `go build ... || touch $@.failed` and touch would fail if the directory was not there. I'll remove.